setup.py

```diff
@@ -178,7 +178,6 @@
             'hepnames2marc = inspirehep.dojson.hepnames2marc:hepnames2marc',
         ],
         'invenio_base.api_apps': [
-            'inspire_theme = inspirehep.modules.theme:INSPIRETheme',
```

Comment:
Ach, that's because we just ask the page to be rendered, but we make no assertion on the data that populates it.